content/sensu-go/5.15/reference/license.md

```diff
@@ -66,12 +66,12 @@ Error: not found
 
 ## Entity limit
 
-Your license includes the entity limit tied to your Sensu licensing package.
-(An entity limit of `0` allows unlimited entities.)
+Your commercial license includes the entity limit tied to your Sensu licensing package.
+An entity limit of `0` allows unlimited entities.
 Both agent and proxy entities count towards the overall entity limit.
-[Contact us](https://account.sensu.io/support) to upgrade your license.
+[Contact us](https://account.sensu.io/support) to upgrade to a commercial license.
```

Comment:
Since it is now a hard entity limit, it still make sense to have the verbiage to upgrade existing licenses as well. In the context of this section of the doc, it is assumed the user already has a license.

Reply:
Thank you -- incorporated!